### PR TITLE
helper PHP class

### PR DESCRIPTION
**Pull request by jszczypk**

### CODE REVIEW

File: src/CountryList.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Umpirsky;
+
+abstract class CountryList
+{
+
```
Comment (umpirsky) [inline src/CountryList.php]:
```suggestion
```

File: src/CountryList.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    static function get(string $language): array
+    {
+	$path = __DIR__.'/../data/'.$language;
+
+	if(!is_dir($path)) {
+	    throw new \Exception("Language $language not found.");
```
Comment (umpirsky) [inline src/CountryList.php]:
```suggestion
	    throw new \RuntimeException("Language $language not found.");
```

File: src/CountryList.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	}
+
+	return require $path.'/country.php';
```
Comment (umpirsky) [inline src/CountryList.php]: We should check if `$path.'/country.php'` exists too.

File: src/CountryList.php [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    }
+
```
Comment (umpirsky) [inline src/CountryList.php]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -42,6 +42,17 @@ Multilingual
 
 All formats are also available in multiple languages, please find full language list [here](https://github.com/umpirsky/country-list/tree/master/data).
 
+Usage
+-----
+
+For PHP
```
Comment (umpirsky) [inline README.md]:
```suggestion
For PHP:
```

File: src/CountryList.php (hunk 2 above)
Comment (jszczypk) [reply src/CountryList.php]:
```suggestion
	    throw new \OutOfBoundsException("Language $language not found.");
```

File: src/CountryList.php (hunk 3 above)
Comment (jszczypk) [reply src/CountryList.php]: Do you want to throw nice exception? `require` will fail anyway.

Comment (umpirsky) [reply src/CountryList.php]: Exception is better.

**Comment by rieschl** [comment]
Hi! Is this going to be merged soon? I'd also need that class :) Thanks!

**Comment by jszczypk** [comment]
I'll try to do it over the weekend.

File: src/CountryList.php (hunk 3 above)
Comment (thistehneisen) [reply src/CountryList.php:15]: Why are you not merging this?

Comment (umpirsky) [reply src/CountryList.php:15]: https://github.com/umpirsky/country-list/pull/99#issuecomment-575591400

**Review by Messhias — APPROVED** [review]

**Comment by Messhias** [comment]
Why this PR wasn't approved yet?

**Comment by umpirsky** [comment]
It should have been closed in favor of https://github.com/umpirsky/country-list/pull/107.

**Pull request closed without merging**